modal_proto/api.proto

```diff
@@ -1227,10 +1228,6 @@ message Image {
   BuildFunction build_function = 21;
 }
 
-message ImageBuilderVersionLookupResponse {
```

Comment:
If you're removing this, you'll have to merge it in after the RPC handler is removed from the server